File: exs/mundo_1/c#/035.cs
using System;

/*
	035 - Analisando triângulo v1.0
    Desenvolva um programa que leia o comprimento de três retas e diga ao usuário se elas podem ou não formar um triângulo
*/		
public class Program
{
	public static void Main()
	{
		int a, b, c;
		bool triangulo = false;
		Console.WriteLine("Digite A: ");
		a = Int32.Parse(Console.ReadLine());
		Console.WriteLine("Digite B: ");
		b = Int32.Parse(Console.ReadLine());
		Console.WriteLine("Digite C: ");
		c = Int32.Parse(Console.ReadLine());
		
		if (a + b > c)
		{
			if (b + c > a)
			{
				if (a + c > b)
				{
					triangulo = true;
				}
			}
		}
		if (triangulo)
		{
			Console.WriteLine("Pode formar um triangulo");
		}
		else
		{
			Console.WriteLine("Não pode formar um triangulo");
		}
	}
}

File: exs/mundo_1/c++/035.cpp
/*
035 - Analisando triângulo v1.0
	Desenvolva um programa que leia o comprimento de três retas e diga
	ao usuário se elas podem ou não formar um triângulo
*/

#include <iostream>

using namespace std;

int main() {
	double r1, r2, r3;

	cin >> r1 >> r2 >> r3;

	if ((r2+r3) > r1 &&
			(r1+r3) > r2 &&
			(r1+r2) > r3) {
		cout << "Forma um triângulo\n";
	} else {
		cout << "Não forma um triângulo\n";
	}
	return 0;
}
